core/lib/multivm/src/versions/vm_latest/tests/nonce_holder.rs

```diff
@@ -101,7 +102,7 @@ fn test_nonce_holder() {
     run_nonce_test(
         1u32,
         NonceHolderTestMode::SetValueUnderNonce,
-        Some("Previous nonce has not been used".to_string()),
+        Some("Error function_selector = 0x13595475, data = 0x13595475".to_string()),
```

Comment:
These tests seem to fail locally. Have you forgot to check in the `contracts` submodule? I see it's present [in the follow-up PR](https://github.com/matter-labs/zksync-era/pull/2967), but not here.